src/core/plugins/json-schema-5/fn.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @prettier
+ */
+import { Map } from "immutable"
+import isPlainObject from "lodash/isPlainObject"
+
+export const schemaHasType = (schema, types) => {
```

Comment:
What would you say to support polymorphic `types`. `types` can either be array or string. If you're looking for a simple "string" you don't need to type ["string"]. It also reflects the nature of JSON Schema 2020-12